### DISCUSSION TITLE
How do I custom import path for prisma client ?

### DISCUSSION THREAD

**Discussion by kittipojsr** [omar-dulaimi/prisma-trpc-generator#18] [Q&A] [answered]
By default prisma-trpc-generator use `import type { Prisma } from '@prisma/client';` for prisma client. But I need to use custom path from [prisma client](https://www.prisma.io/docs/concepts/components/prisma-client/working-with-prismaclient/generating-prisma-client#the-location-of-prisma-client) generator schema output.

**Comment by omar-dulaimi** [comment]
Issue for ref: https://github.com/omar-dulaimi/prisma-trpc-generator/issues/19

**Comment by omar-dulaimi** [comment, marked as answer]
Released in https://github.com/omar-dulaimi/prisma-trpc-generator/releases/tag/0.4.0

Enjoy :)
